quirks/password-rules.json

```diff
@@ -227,6 +227,9 @@
     },
     "idestination.info": {
         "password-rules": "maxlength: 15;"
+    },
+     "impots.gouv.fr": {
+        "password-rules": "required: digit; minlength:12; maxlength: 128; allowed : [!#$%&*+-/=?^_'.{|}];"
```

Comment:
I think you want:

```
minlength: 12; maxlength: 128; required: digit; allowed: [-!#$%&*+/=?^_'.{|}];
```

Fixed up some spacing, and moved `-` to be the first character of the character class.